Detector/DetFCChhCalDiscs/compact/Forward_coneCryo.xml

```diff
@@ -57,19 +57,19 @@
   </define>
   <readouts>
     <readout name="EMFwdPhiEta">
-      <segmentation type="FCCSWGridPhiEta" phi_bins="256" offset_phi="-pi+(pi/256.)" grid_size_eta="0.025" offset_eta="-5.9975"/>
+      <segmentation type="FCCSWGridPhiEta" phi_bins="256" offset_phi="-pi+(pi/256.)" grid_size_eta="0.025" offset_eta="-6.00024"/>
```

Comment:
see question above.

Reply:
again - consistency with ecal barrel. Otherwise we need to be careful to always run sim and reco on the same set of calorimeters.